datastore/gcloud/rest/datastore/value.py

```diff
@@ -89,6 +92,7 @@ def _get_supported_types(cls):
         # type: () -> Dict[type, TypeName]
         supported_types = TYPES
         supported_types.update({
+            LatLng: TypeName.GEOPOINT,
```

Comment:
Does this need to be dynamically added here or should we just throw it in `TYPES`?

Reply:
I just figured that it would make `constants.py` depend on the rest of the package, which is not the case at the moment.
I must admit I haven't even tried to add it to `TYPES`, assuming it would create a circular dependency. I'll have another look.